pipeline/metadata/design_table.csv

```diff
@@ -2399,35 +2399,35 @@ PNNL_33,sapflow,Format,TMP,S,Sapflow,33,,,,,
 PNNL_33,sapflow,RECORD,TMP,S,Sapflow,33,,,,,
 PNNL_33,sapflow,Statname,TMP,S,Sapflow,33,,,,,
 PNNL_33,sapflow,BattV_Avg,TMP,S,Sapflow,33,,,,battery_voltage,
-PNNL_33,sapflow,"DiffVolt_Avg({1,2,3,4,5,6,12})",TMP,S,Sapflow,33,"{S5,S6,S9,S10,S12,S14,S19}","{F5,G6,F2,G5,F4,I2,D2}",,sapflow_2.5cm,
-PNNL_33,sapflow,"DiffVolt_Avg({9,10})",TMP,S,Sapflow,33,"{S14,S12}D","{I2,F4}",,sapflow_5cm,
+PNNL_33,sapflow,"DiffVolt_Avg({1,2,3,4,5,6,12})",TMP,S,Sapflow,33,"{S5,S6,S9,S10,S12,S14,S19}",,,sapflow_2.5cm,
+PNNL_33,sapflow,"DiffVolt_Avg({9,10})",TMP,S,Sapflow,33,"{S14,S12}D",,,sapflow_5cm,
 PNNL_33,sapflow,"DiffVolt_Avg({7,8,11,13,14})",TMP,S,Sapflow,33,,,,,
 PNNL_33,sapflow,DiffVolt({1:14}),TMP,S,Sapflow,33,,,,,
 ,,,,,,,,,,,
 PNNL_33,Terosdata,Format,TMP,S,TEROS12,33,,,,,
 PNNL_33,Terosdata,RECORD,TMP,S,TEROS12,33,,,,,
 PNNL_33,Terosdata,Statname,TMP,S,TEROS12,33,,,,,
-PNNL_33,Terosdata,"Teros({1:9},1)",TMP,S,TEROS12,33,"{T167,T165,T065,T161,T067,T068,T069,T070,T071}","{F3,G3,I3,G4,H4,F5,G5,H5,I5}",,soil_vwc_15cm,
-PNNL_33,Terosdata,"Teros({1:9},2)",TMP,S,TEROS12,33,"{T167,T165,T065,T161,T067,T068,T069,T070,T071}","{F3,G3,I3,G4,H4,F5,G5,H5,I5}",,soil_temp_15cm,
-PNNL_33,Terosdata,"Teros({1:9},3)",TMP,S,TEROS12,33,"{T167,T165,T065,T161,T067,T068,T069,T070,T071}","{F3,G3,I3,G4,H4,F5,G5,H5,I5}",,soil_EC_15cm,
+PNNL_33,Terosdata,"Teros({1:9},1)",TMP,S,TEROS12,33,"{T167,T165,T065,T161,T067,T068,T069,T070,T071}",,,soil_vwc_15cm,
+PNNL_33,Terosdata,"Teros({1:9},2)",TMP,S,TEROS12,33,"{T167,T165,T065,T161,T067,T068,T069,T070,T071}",,,soil_temp_15cm,
+PNNL_33,Terosdata,"Teros({1:9},3)",TMP,S,TEROS12,33,"{T167,T165,T065,T161,T067,T068,T069,T070,T071}",,,soil_EC_15cm,
 PNNL_33,Terosdata,"Teros({10:11},1)",TMP,S,TEROS12,33,,,,,
 PNNL_33,Terosdata,"Teros({10:11},2)",TMP,S,TEROS12,33,,,,,
 PNNL_33,Terosdata,"Teros({10:11},3)",TMP,S,TEROS12,33,,,,,
-PNNL_33,Terosdata,"Teros({12:13},1)",TMP,S,TEROS12,33,"T0{87,88}","{F2,H2}",,soil_vwc_15cm,
-PNNL_33,Terosdata,"Teros({12:13},2)",TMP,S,TEROS12,33,"T0{87,88}","{F2,H2}",,soil_temp_15cm,
-PNNL_33,Terosdata,"Teros({12:13},3)",TMP,S,TEROS12,33,"T0{87,88}","{F2,H2}",,soil_EC_15cm,
+PNNL_33,Terosdata,"Teros({12:13},1)",TMP,S,TEROS12,33,"T0{87,88}",,,soil_vwc_15cm,
+PNNL_33,Terosdata,"Teros({12:13},2)",TMP,S,TEROS12,33,"T0{87,88}",,,soil_temp_15cm,
+PNNL_33,Terosdata,"Teros({12:13},3)",TMP,S,TEROS12,33,"T0{87,88}",,,soil_EC_15cm,
 PNNL_33,Terosdata,"Teros({14:16},1)",TMP,S,TEROS12,33,,,,,
 PNNL_33,Terosdata,"Teros({14:16},2)",TMP,S,TEROS12,33,,,,,
 PNNL_33,Terosdata,"Teros({14:16},3)",TMP,S,TEROS12,33,,,,,
-PNNL_33,Terosdata,"Teros({17,20},1)",TMP,S,TEROS12,33,"T{092,046}","{I3,G5}",,soil_vwc_5cm,
```

Comment:
@stephpenn1 Why did you make these changes throughout the TEMPEST mappings? 😕

Reply:
Not sure what happened here - I used [this code](https://github.com/COMPASS-DOE/sensor-data-pipeline/blob/ow-plot/pipeline/metadata/edit-design-table.R) which should only edit ExoTable rows

Reply:
Weird!  _Something_ certainly went sideways in b4a09b459770227fe79897a654f51d5f827ba902 🤷